blog/_posts/2017-09-18-story-of-an-issue.md

```diff
@@ -0,0 +1,100 @@
+
+For instance, the more back-end the better. We just love our RoR programming! Of course we also love it when we learn and understand front-end code, but it's very clear that we both are more attracted to the back-end side.
+
+#### Ask the diaspora\* community
+
+Diaspora\* has a very active community and we had to find out for each issue we chose if it was still available. We got great feedback from the community just by asking about it on the issue page. In [some cases](https://github.com/diaspora/diaspora/issues/7471) it even triggered a long discussion about ways to solve the issue or even the very necessity of a solution. The community is very responsive whenever we have questions or require consultation on our solution. You just gotta love the diaspora\* community for their involvement and investment.
+
+#### Search. Everywhere.
+
+Like two Alices in diasporaland, with every issue we embark on an ongoing quest through the intricate and entangled maze called the diaspora source code. It is not your typical MVC app (if that even really exists). Finding the related files to the issue and trying to understand the relationship between them is no less than a superpower one needs to develop.
+
+Apart from the familiar models, controllers and views, we have encountered, and thereby learned about, lots of other types of files: mailers, presenters, helpers, services, workers, templates, javascript files, specs, factories and more.  
+<div style="text-align:center"><img src ="img/blog/2017/2017-09-18-codebears1.gif" style="width: 350px;" /></div>
+
+#### Tutorials, Tutorials, Tutorials
+
+In order to understand all the new terms we learn, we spend quite a lot of time watching and reading tutorials (We find [Lynda[](https://www.lynda.com/), among others, to be an excellent source for beginners tutorials). Overwhelmed by all this new theory, our coaches are very helpful with mediating and explaining new and complicated concepts.
+
+#### Get confused
+
+Connecting theory and practice always, but always, resulted in a big confusion. Things on diaspora almost never work according to theory. We think we figured out where we need to add code, it fails, the error raises some filename we don't know, which leads us to go back to our diasporaland quest, where we don't understand the code, or we find a bunch of new terms, or we can't figure out why this file even exists, and so on.
+
+The most common sentence called out loud during this summer, by students and coaches alike, while looking at the code is: "This is weird".      
+<div style="text-align:center"><img src ="img/blog/2017/2017-09-18-codebears2.gif" /></div>
+
+#### Play with the code
+
+Puzzled, with no other choice, we play with the code and see what happens. After realizing we can't really ruin anything as long as we work on our local branch, anything goes.
```

Comment:
This sentence looks bit odd. Did you mean to add some text after 'anything goes'?